packages/neuron-ui/src/components/MultisigAddress/index.tsx

```diff
@@ -296,7 +299,7 @@ const MultisigAddress = () => {
                                   key={key}
                                   data-key={key}
                                   onClick={onClickItem(item)}
-                                  disabled={disabled}
+                                  disabled={key === ApproveKey ? !canSign || disabled : disabled}
```

Comment:
Availability should be set in the action list at https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-ui/src/components/MultisigAddress/index.tsx#L120

```ts
        disabled: selectIds.length > 1 && item.key !== 'delete',
```

Reply:
It depends on the data source, so it can not set value there.

Reply:
> It depends on the data source, so it can not set value there.

The available address list can be a condition of `listActionOptions`, or to say, `listActionOptions` changes when the address list changes

Reply:
`listActionOptions` is a global option for every item. But every item's `Approve` disable status may be different.

Reply:
got it